pkg/limayaml/limayaml.go

```diff
@@ -98,6 +98,7 @@ type VMOpts struct {
 
 type QEMUOpts struct {
 	MinimumVersion *string `yaml:"minimumVersion,omitempty" json:"minimumVersion,omitempty" jsonschema:"nullable"`
+	VirtioGA       *bool   `yaml:"virtioGA,omitempty" json:"virtioGA,omitempty" jsonschema:"nullable"`
```

Comment:
Maybe this should be `guestAgentTransportType *string` ?
Also, this may potentially apply to non-QEMU VM drivers?

Reply:
Good call! I will check if this could be applicable for other platforms. Having it as typed option instead of boolean flag feels like a better design.